src/index.html

```diff
@@ -6,13 +6,60 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Product card</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main>
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          class="product-card__img"
+          src="images/imac.jpeg"
+          alt="Product image"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">Product code: 195434</p>
+        <div class="product-card__review">
+          <div class="product-card__stars">
+            <div class="product-card__stars-star"></div>
+            <div class="product-card__stars-star"></div>
+            <div class="product-card__stars-star"></div>
+            <div class="product-card__stars-star"></div>
+            <div class="product-card__stars-star"></div>
```

Comment:
The class names for the stars suggest that you might use BEM methodology incorrectly. According to BEM, the class should be `product-card__star` instead of `product-card__stars-star`. Also, if you are using BEM, you should not create a separate block for stars, but rather treat each star as an element of the `product-card__stars` block.